### DISCUSSION TITLE
How can I increase speed of checkout process?

### DISCUSSION THREAD

**Discussion by jaxkeiser** [btcpayserver/btcpayserver#4280] [Technical Support]
I setup BTCPay Server with myNode for my Woocommerce store using this guide: https://mynodebtc.github.io/btcpay-server/setup.html and I got the https certificate using lets encrypt.

I'm wondering all the ways in which I can increase the speed at which my ecommerce store redirects customers to my BTCPay Server checkout screen when customers click pay with BTCPay.

For example, I was able to drastically increase the speed by disabling Tor and I2P for my node within the myNode control panel.

Is there anything else?

**Comment by pavlenex** [comment]
I think this is something you'd have to bring up to the MyNode community, as they're using a custom deployment method afair. For best performance I would recommend running btcpay in a VPS.

**Reply by dennisreimann** [reply]
I'd also recomment going with the VPS option, because with the node at home setup you will be limited by the bandwidth of your home internet connection and also leak your IP address.
